skylib/kustomize/kustomize.bzl

```diff
@@ -234,6 +234,11 @@ def _kustomize_impl(ctx):
                 if "{" in regrepo:
                     regrepo = stamp(ctx, regrepo, tmpfiles, ctx.attr.name + regrepo.replace("/", "_"))
                 template_part += " --variable={}={}@$(cat {})".format(kpi.image_label, regrepo, kpi.digestfile.path)
+
+                # Image hash
+                template_part += " --variable={}=$(cat {} | awk '{{print substr($1,8)}}')".format(str(kpi.image_label)+"__HASH", kpi.digestfile.path)
```

Comment:
To keep the script more portable, I'd go about without using `awk`.

Here is the same thing done in push rule: https://github.com/adobe/rules_gitops/blob/master/skylib/push.bzl#L97
```
tag = "$(cat {} | cut -d ':' -f 2 | cut -c 1-7)".format(_get_runfile_path(ctx, ctx.outputs.digest))
```

Reply:
Sure. I will update it to use `cut`. I myself was debating which one is more portable `cut` vs. `awk`.